### PR TITLE
Added some what's new information about #2435 and #2810.

### PR DESCRIPTION
**Pull request by pelson**
I'm aware that it is very terse - it will need editorial along with the rest of the "what's new" items.

### CODE REVIEW

File: docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-27_grib_removed.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+* The deprecated :mod:`iris.fileformats.grib` was removed. All iris GRIB functionality is now delivered through :mod:`iris-grib`.
```
Comment (dkillick) [inline docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-27_grib_removed.txt]: "iris" --> "Iris".

File: docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-27_logging_removed.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+* In Iris v1 it was possible to configure Iris to log at import time through :attr:`iris.config.LOGGING`. This capability has been removed in iris 2.
```
Comment (dkillick) [inline docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-27_logging_removed.txt]: "to log at import time": to log _what_ at import time?

"iris 2" --> "Iris v2"

**Comment by dkillick** [comment]
@pelson I think a rebase might solve the failing tests...

**Comment by pelson** [comment]
That should do it...

**Pull request merged**